src/Module.php

```diff
@@ -89,7 +89,7 @@ function ($services) {
                         /** @var HydratorPluginManager $hydrators */
                         $hydrators = $services->get('HydratorManager');
                     } else {
-                        $hydrators = new HydratorPluginManager();
+                        $hydrators = new HydratorPluginManager($services);
```

Comment:
Plugin managers require a container to be passed (throw a deprecation notice if not passed), trying to do this wherever possible.